### DISCUSSION TITLE
Blog 7

### DISCUSSION THREAD

**Discussion by JustMischka** [Anastylos/Jump-and-Run#77] [Blog]
# Qualitätsmerkmale #

Um ein robustes und nachhaltiges System zu entwickeln, müssen die Qualitätsmerkmale in greifbare Szenarien übersetzt werden. Die Methode, die wir verwenden, basiert auf dem Schema „Quelle-Auslöser-Artefakt-Umgebung-Antwort-Antwortmaß“.

![2024-11-22_Qualitätsbaum drawio](https://github.com/user-attachments/assets/4766508e-39e8-46e8-bbfa-73160f29c1e3)

## Qualitätsbaum als Leitfaden: ##
Der gezeigte Qualitätsbaum dient hierbei als zentraler Leitfaden, um jedem Qualitätsmerkmal klar zugeordnete Szenarien zu identifizieren:

### Spaßfaktor: ### 
Szenarien wie A10, A20, A30 konzentrieren sich auf Benutzererfahrungen wie Atmosphäre, Abwechslung und intuitives Design. Ziel ist es, ein immersives und ansprechendes Nutzungserlebnis zu schaffen.

### Support: ### 
Szenarien B10 und B20 behandeln Aspekte wie die Zugänglichkeit eines Bugreport-Systems und effektive Kontaktwege für Support.

### Verfügbarkeit: ### 
Szenarien wie C10 (Distributionsplattform) und C20 (Hybride Eingabetypen) fokussieren auf die technische Zugänglichkeit.

### Erweiterbarkeit: ### 
DLC und Customisation (D10, D20) fördern die Anpassungsfähigkeit des Systems an Benutzerwünsche.

### Wartbarkeit: ### 
Eine klare Struktur (E10) und einheitliche Tools (E20) sollen die langfristige Pflege und Anpassung des Systems erleichtern.

## Strategien und Priorisierung ##
In jedem Projekt ist es essenziell, Ressourcen effektiv einzusetzen. Deshalb ist es ratsam, sich auf ein bis zwei Szenarien mit hohen Prioritäten zu konzentrieren.

### Hohes Prioritätsszenario 1: A30 - Intuitives Design ###
Benutzerfreundlichkeit ist ein zentraler Faktor für den Erfolg eines Spiels. Ein ansprechendes und leicht zugängliches Design sorgt dafür, dass die Spieler sich schnell zurechtfinden und eine positive Spielerfahrung haben.

### Hohes Prioritätsszenario 2: C10 - Distributionsplattform ###
Die Wahl einer geeigneten Plattform wie Steam oder Epic Games Store ist essenziell, um das Spiel einem breiten Publikum zugänglich zu machen. Da es sich um ein reines PC-Spiel handelt, sollte der Fokus auf Plattformen mit einer hohen Reichweite und gutem Community-Support liegen.

### Steam Direct oder Epic Games Publishing: ###
Registrierung und Nutzung der jeweiligen Dienste zur Distribution. Beide Plattformen bieten Features wie automatische Updates, Community-Integration (z. B. Foren, Rezensionen) und Vermarktungsmöglichkeiten.

### Community-Building auf Plattformen: ###
Aufbau einer Spielergemeinschaft durch Foren, Beta-Tests und Feedback-Kanäle innerhalb der Distributionsplattformen.

### Vermarktung und Sichtbarkeit: ###
Erstellung ansprechender Shop-Seiten mit aussagekräftigen Screenshots, Trailern und einer überzeugenden Beschreibung, um die Sichtbarkeit des Spiels zu erhöhen.

**Comment by MB-stack06** [comment]
Dieser Beitrag hat das gesamte Team sehr inspiriert und viele wertvolle Impulse geliefert! 
Die detaillierte Betrachtung der Qualitätsmerkmale, die ein Spiel wirklich erfolgreich und nachhaltig machen, ist äußerst hilfreich. Besonders der Fokus auf die Benutzererfahrung und die Wahl der richtigen Plattform finden wir sehr durchdacht. 
Auch die Vorschläge zur Communitybildung und zur langfristigen Planung zeigen eine klare Vision, wie man das Projekt weiter voranbringen kann. 

Wir sind begeistert von der durchdachten Herangehensweise und freuen uns auf die weiteren Schritte. 
Ein großes Kompliment an euer gesamtes Team.

Euer GymTrackerX-Team

**Comment by OmarMoujahed** [comment]
Der Qualitätsbaum bietet eine klare und übersichtliche Struktur zur Identifikation und Priorisierung von Qualitätsmerkmalen. Besonders spannend finde ich die Verbindung von intuitivem Design und Community-Building als Schlüsselstrategien für nachhaltigen Erfolg. Sehr inspirierend und praxisnah formuliert!

Mit freundlichen Grüßen
Team HÆLTH
